.github/workflows/label_merge_conflicts.yml

```diff
@@ -0,0 +1,11 @@
+name: 'Check for merge conflicts'
+on:
+  push
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mschilde/auto-label-merge-conflicts@master
```

Comment:
```suggestion
      - uses: mschilde/auto-label-merge-conflicts@v2.0
```
Same as above.

Reply:
Nevermind, v2.0 does not have a feature we want https://github.com/mschilde/auto-label-merge-conflicts/commit/f842c977cada4f2f391aa98bfbe126e24ccedd57, so master should be fine for now.